CHANGELOG.md

```diff
@@ -2,9 +2,36 @@
 
 ## [Unreleased]
 
-* Export the `DEFAULT_COLUMN_FAMILY_NAME` constant.
-* Implement `Clone` trait for `Options`.
-* Added `Send` implementation to `WriteBatch`.
+## 0.15.0 (2020-08-??)
+
+* Fix building rocksdb library on windows host (aleksuss)
+* Add github actions CI for windows build (aleksuss)
+* Update doc for Options::set_compression_type (wqfish)
+* Add clippy linter in CI (aleksuss)
+* Use DBPath for backup_restore test (wqfish)
+* Allow to build RocksDB with a different stdlib (calavera)
+* Add some doc-comments and tiny refactoring (aleksuss)
+* Expose `open_with_ttl`. (calavera)
+* Fixed build for `x86_64-linux-android` that doesn't support PCLMUL (vimmerru)
+* Add support for `SstFileWriter` and `DB::ingest_external_file` (methyl)
+* Add set_max_log_file_size and set_recycle_log_file_num to the Options (stanislav-tkach)
+* Export the `DEFAULT_COLUMN_FAMILY_NAME` constant (stanislav-tkach)
+* Fix slice transformers with no in_domain callback (nelhage)
+* Don't segfault on failed a merge operator (nelhage)
+* Adding read/write/db/compaction options (linxGnu)
+* Add dbpath and env options (linxGnu)
+* Add compaction filter factory API (unrealhoang)
+* Add link stdlib when linking prebuilt rocksdb (unrealhoang)
+* Remove allow(clippy::wildcard_imports) (stanislav-tkach)
```

Comment:
Should we mention this change? It was just a temporary workaround and it doesn't affect library clients.

Reply:
Removed